examples/tensorflow/image_recognition/tensorflow_models/mobilenet_v2/export/requirements.txt

```diff
@@ -1,6 +1,6 @@
-tensorflow>=2.11.1
-intel-extension-for-tensorflow[cpu]
-tf2onnx>=1.10.1
-onnx
-onnxruntime
-onnxruntime-extensions; python_version < '3.11'
+tensorflow==2.11.1
+intel-extension-for-tensorflow[cpu]==1.1.0
+tf2onnx==1.14.0
```

Comment:
have to use certain version or can be >= certain version?

Reply:
Actually, I have fixed similar issue of tf2onnx example by using ">=" version two weeks ago(PR1131). 
But dependency issue occurs again because of version updates of some pakages.
Therefore, I think we'd better to make it fixed.